src/main/java/de/mediathekview/mserver/crawler/rbb/tasks/RbbTopicOverviewTask.java

```diff
@@ -0,0 +1,75 @@
+package de.mediathekview.mserver.crawler.rbb.tasks;
+
+import static de.mediathekview.mserver.base.Consts.ATTRIBUTE_HREF;
+
+import de.mediathekview.mserver.base.utils.UrlUtils;
+import de.mediathekview.mserver.crawler.basic.AbstractCrawler;
+import de.mediathekview.mserver.crawler.basic.AbstractDocumentTask;
+import de.mediathekview.mserver.crawler.basic.AbstractRecrusivConverterTask;
+import de.mediathekview.mserver.crawler.basic.CrawlerUrlDTO;
+import de.mediathekview.mserver.crawler.rbb.RbbConstants;
+import java.util.concurrent.ConcurrentLinkedQueue;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+public class RbbTopicOverviewTask extends AbstractDocumentTask<CrawlerUrlDTO, CrawlerUrlDTO> {
+
+  private static final String ATTRIBUTE_DISABLED = "aria-disabled";
+
+  private static final String SELECTOR_NEXT_PAGE = "div.entries > div.entry > a[aria-labelledby=cursorRight]";
+  private static final String SELECTOR_TOPIC_ENTRY = "div.elementWrapper div.teaser > div.textWrapper > a.textLink";
+
+  private final int pageNumber;
+
+  public RbbTopicOverviewTask(AbstractCrawler aCrawler,
+      ConcurrentLinkedQueue<CrawlerUrlDTO> aUrlToCrawlDtos) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') The type of the "aUrlToCrawlDtos" object should be an interface such as "Queue" rather than the implementation "ConcurrentLinkedQueue". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1319)